docs/source/Introduction.rst

```diff
@@ -1,2 +1,106 @@
+
+Here's an example of how to build an Nginx web server on top of a Debian base image using the Dockerfile maintained by Nginx and published in GitHub::
+
+    podman build -t nginx https://git.io/Jf8ol
+
+Once, the image build completes, it’s easy to run the new image from our local cache::
+
+    podman run -d -p 8080:80 nginx
+    curl localhost
+
+Output::
+
+    ...
+    <p><em>Thank you for using nginx.</em></p>
+    ...
+
+Building new images is great, but sharing our work with others let’s them review our work, critique how we built them, and offer improved versions. Our newly built Nginx image could be published at quay.io or docker.io to share it with the world. Everything needed to run the Nginx application is provided in the container image. Others could easily pull it down and use it, or make improvements to it.
+
+Standardizing on container images and `Container Registries`_ enable a new level of collaboration through simple consumption. This simple consumption model is possible because every major Container Engine and Registry Server uses the Open Containers Initiative (OCI_) format. This allows users to find, run, build, share and deploy containers anywhere they want. Podman and other `Container Engines`_ like CRI-O, Docker, or containerd can create and consume container images from docker.io, quay.io, an on premise registry or even one provided by a cloud provider. The OCI image format facilitates this ecosystem through a single standard.
+
+For example, if we wanted to share our newly built Nginx container image on quay.io it’s easy. First log in to quay::
+
+    podman login quay.io
+Input::
+
+    Username: USERNAME
+    Password: ********
+    Login Succeeded!
+
+Nex, tag the image so that we can push it into our user account::
+
+    podman tag localhost/nginx quay.io/USERNAME/nginx
+
+Finally, push the image::
+
+    podman push quay.io/USERNAME/nginx
+
+Output::
+
+    Getting image source signatures
+    Copying blob 38c40d6c2c85 done
+    Copying blob fee76a531659 done
+    Copying blob c2adabaecedb done
+    Copying config 7f3589c0b8 done
+    Writing manifest to image destination
+    Copying config 7f3589c0b8 done
+    Writing manifest to image destination
+    Storing signatures
+
+Notice that we pushed four layers to our registry and now it’s available for others to share. Take a quick look::
+
+    podman inspect quay.io/fatherlinux/nginx
+
+Output::
+
+    [
+        {
+            "Id": "7f3589c0b8849a9e1ff52ceb0fcea2390e2731db9d1a7358c2f5fad216a48263",
+            "Digest": "sha256:7822b5ba4c2eaabdd0ff3812277cfafa8a25527d1e234be028ed381a43ad5498",
+            "RepoTags": [
+                "quay.io/USERNAME/nginx:latest",
+    ...
+
+To summarize, Podman makes it easy to find, run, build and share containers.
+
+* Find: whether finding a container on dockerhub.io or quay.io, an internal registry server, or directly from a vendor a couple of find, or pull commands make it easy
```

Comment:
Having a hard time with this one as we don't have a "find" command and pull doesn't help much with searching  unless you know the image.  Maybe work in podman search instead?  http://docs.podman.io/en/latest/markdown/podman-search.1.html

Reply:
Fixed. I made it more explicit as "podman search" and "podman pull"